snap/snapcraft.yaml

```diff
@@ -59,6 +59,16 @@ slots:
     source:
       read:
         - $SNAP_COMMON/var/log/mysql
+  mysql-sockets:
+    interface: content
+    content: socket-directory
+    write:
+      - $SNAP_COMMON/var/run/mysqld
```

Comment:
JFMI, why do we expose mysql-socket? It is not a subordinate charm.
I see only one use case: someone install snap directly and want to connect mysql from another locally installed snap. Is it your usecase?

Reply:
You bring up a good point, I had not given much thought to exposing mysql sockets. Should we remove this chunk of code which exposes them?

Reply:
Removed mysql-sockets slot in f8dfccff25fa8204866cb201cfb3a3b2cafe7b35